Classes/Database/TestCase.php

```diff
@@ -38,12 +46,12 @@ public function __construct($name = null, array $data = [], $dataName = '')
         /** @var array $databaseConfiguration */
         $databaseConfiguration = $GLOBALS['TYPO3_CONF_VARS']['DB'];
         if (\TYPO3\CMS\Core\Utility\VersionNumberUtility::convertVersionNumberToInteger(TYPO3_version) >= 8001000) {
-            $originalDatabaseName = $databaseConfiguration['Connections']['Default']['dbname'];
+            $this->originalDatabaseName = $databaseConfiguration['Connections']['Default']['dbname'];
```

Comment:
We need to think about the gettig of dbname and properties in general, `Default` is not always the connection name